release-plz.toml

```diff
@@ -0,0 +1,55 @@
+[changelog]
+body = """
+
+## [{{ version | trim_start_matches(pat="v") }}]{%- if release_link -%}({{ release_link }}){% endif %}
+_{{ timestamp | date(format="%d %B %Y") }}_
+{% for group, commits in commits | group_by(attribute="group") -%}
+{%- if group != "chore" %}
+### {{ group | upper_first }}
+
+{% for commit in commits -%}
+{%- if commit.scope and commit.scope != package -%}
+* *({{commit.scope}})* {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message | upper_first }}{%- if commit.links %} ({% for link in commit.links %}[{{link.text}}]({{link.href}}) {% endfor -%}){% endif %}
+{% else -%}
+* {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message | upper_first }}
+{% endif -%}
+{%- endfor %}
+
+{%- endif %}
+
+{%- endfor -%}
+"""
+
+commit_parsers = [
+  { message = "^feat", group = "added" },
+  { message = "^changed", group = "changed" },
+  { message = "^deprecated", group = "deprecated" },
+  { message = "^fix", group = "fixed" },
+  { message = "^security", group = "security" },
+  { message = "^chore", group = "chore" },
+  { message = "^.*", group = "other" },
+]
+
+[workspace]
+dependencies_update = true
+features_always_increment_minor = true
+pr_labels = ["release"]
+release_always = false
+release_commits = "^(feat|fix)[(:]"
+
+[[package]]
+name = "c2pa"
+changelog_path = "./CHANGELOG.md"
```

Comment:
Without the `changelog_path` entry, release-plz would want to put the changelog for `c2pa` at `sdk/CHANGELOG.md`. IMHO we probably want to keep it where it is, thus the override.